tests/tests.go

```diff
@@ -331,6 +335,14 @@ func addLabelsToNamespace(namespace string, labels map[string]string) error {
 	return err
 }
 
+func getEnv(key, fallback string) string {
```

Comment:
I think we should remove `fallback` parameter and initialize [this](https://github.com/k8snetworkplumbingwg/kubemacpool/pull/145/commits/74ecb40c55a63216809d28e1fa17f17714e2a64c#diff-4de094b4f2274c29e9f11102da4ca7d0R39) with the `names.MANAGER_NAMESAPCE

Reply:
This is a generic getEnv function, we don't want to harcode it here, or do you mean to do getManagerNamespace function ?